app/views/eventOverview.html

```diff
@@ -135,5 +135,73 @@ <h2 class="page-title border" translate>Links</h2>
         </div>
       </div>
     </section>
+    <section>
+      <div
+        class="alert"
+        ng-class="notify.class"
+        ng-bind-html="notify.message"
+        ng-show="notify.class"
+      ></div>
+
+      <h2 class="page-title border" translate>Image</h2>
+      <div class="row">
+        <div class="col-sm-5">
+          <label translate for="fileInput">Select an image file: </label>
+
+          <input
+            read-file
+            type="file"
+            accept="image/*"
+            id="fileInput"
+            class="form-control-file"
+            ng-model="selectedImage"
+          />
+          <p></p>
```

Comment:
Is this empty <p> just for spacing purposes?

Reply:
removed